tests/test_logging.py

```diff
@@ -14,6 +19,19 @@ def has_handler(logger, handler_name):
     return any([handler.name == handler_name for handler in logger.handlers])
 
 
+def capture_stdout(command, *args, **kwargs):
```

Comment:
Did you consider using a contextwrapper instead? IIRC there's one in the stdlib unittest for capturing the stdout/stderr.